ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -29,38 +48,52 @@ const predictionFieldValue = (
 
   const infoValue = predictions[infoField]
 
-  if (dangerThreshold) {
-    value = parseFloat(value).toPrecision(2)
-    let color = 'green'
-    if (value >= dangerThreshold) {
-      color = 'red'
-    } else if (value >= warningThreshold) {
-      color = 'yellow'
-    }
-    return { value, color, infoValue, infoTitle, dangerThreshold, warningThreshold }
+  if (thresholds) {
+    value = parseFloat(value).toPrecision(3)
+    const color = PRED_COLOR_MAP.find((clr, i) => comparePathScores(value, i, thresholds))
+    return { value, color, infoValue, infoTitle, thresholds }
   }
 
   return indicatorMap[value[0]] || indicatorMap[value]
 }
 
+const coloredIcon = color => <ColoredIcon name="circle" size="small" color={color} />
+
 const Prediction = (
-  { field, fieldTitle, value, color, infoValue, infoTitle, warningThreshold, dangerThreshold, href },
+  { field, fieldTitle, value, color, infoValue, infoTitle, thresholds, href },
 ) => {
   const indicator = infoValue ? (
     <Popup
       header={infoTitle}
       content={infoValue}
       trigger={<Icon name="question circle" size="small" color={color} />}
     />
-  ) : <Icon name="circle" size="small" color={color} />
+  ) : coloredIcon(color)
   const fieldName = fieldTitle || snakecaseToTitlecase(field)
-  const fieldDisplay = dangerThreshold ? (
+  const fieldDisplay = thresholds ? (
     <Popup
       header={`${fieldName} Color Ranges`}
       content={
         <div>
-          <div>{`Red > ${dangerThreshold}`}</div>
-          {warningThreshold < dangerThreshold && <div>{`Yellow > ${warningThreshold}`}</div>}
+          {[0, 1].map(i => thresholds[i] !== undefined && (
+            <div>
+              {coloredIcon(PRED_COLOR_MAP[i])}
+              {i > 0 && thresholds[i - 1] !== undefined && ` > ${thresholds[i - 1]} and`}
+              {` <= ${thresholds[i]}`}
+            </div>
+          ))}
+          <div>
+            {coloredIcon(PRED_COLOR_MAP[2])}
+            {thresholds[1] === undefined ? '' : ` > ${thresholds[1]} and`}
+            {` < ${thresholds[2]}`}
+          </div>
+          {[2, 3, 4].map(i => thresholds[i] !== undefined && (
+            <div>
+              {coloredIcon(PRED_COLOR_MAP[i + 1])}
+              {` >= ${thresholds[i]}`}
+              {i < 4 && thresholds[i + 1] !== undefined && ` and < ${thresholds[i + 1]}`}
+            </div>
+          ))}
```

Comment:
The UI looks like the following picture.
![image](https://github.com/broadinstitute/seqr/assets/33550059/5e88d1d8-b5f4-496b-954d-c87c680b76d2)

Reply:
Just show "color > threshold" for everyhting except the first score which shows <=. DO not show both boundry numbers for each color, just the lower boundry

Reply:
How about we do it like the following?
```
<colored dot> <= 0.0016
<colored dot> <= 0.29
<colored dot> < 0.644
<colored dot> < 0.773
<colored dot> < 0.932
<colored dot> >= 0.932
```

Reply:
I'm concerned about the mixing of `<=` and `<`, we need to be consistent in setting our thresholds so we always use the same comparator. Otherwise this is fine, having the final one be `>=` works

Reply:
The lower color ranges open on the left and close on the right. The upper ones close at the left and open at the right. That's why we have both `<=` and `<`.

Reply:
And what I am saying is we should define all the thresholds so they have a consistant cutoff. Wither everything should be strict less than, or <=, you define the threshhol so you can change it if you need to to get this consistancy

Reply:
In the example you gave, this is the functional equivalent:
```
<colored dot> < 0.0017
<colored dot> < 0.2901
<colored dot> < 0.644
<colored dot> < 0.773
<colored dot> < 0.932
<colored dot> >= 0.932
```

Reply:
The code will be much easier. But the thresholds are slightly different from the provided numbers. Is it okay for the analysts?

Reply:
You should also be doing this in a single list iteration. The logic is not 100% correct here but something like this will work:
```
thresholds.map((threshold, i) => {
  if (threshold === undefined && thresholds[i+1] === undefined) {
    return null
  }
  return (
      <div>
          {coloredIcon(PRED_COLOR_MAP[i])}
          {thresholds[i+1] !== undefined ? ` < ${thresholds[i+1]}` : ` >= ${threshold}` }
       </div>
  )
})
```